module/reducer.js

```diff
@@ -30,7 +34,7 @@ export function createExperiments (experiments) {
 			case 'LOAD_EXPERIMENTS_VARIANTS': {
 				const newState = {}
 				for (let key in state) {
-					if (action.state.hasOwnProperty(key) && state.hasOwnProperty(key)) {
+					if (action.state.hasOwnProperty(key) && state.hasOwnProperty(key) && findVariant(experiments[key], action.state[key])) {
```

Comment:
nipick - I think you should put `findVariant(experiments[key], action.state[key])` in a separate if statement

Reply:
ok, added